tests/integration/dependencies_test.go

```diff
@@ -65,9 +65,7 @@ func installWordpressBundle(p *porter.TestPorter) (namespace string) {
 	installOpts.Params = []string{
 		"wordpress-password=mypassword",
 		"namespace=" + namespace,
-		"wordpress-name=porter-ci-wordpress-" + namespace,
```

Comment:
Since we are installing into a unique namespace, we don't need to make the release name unique. This was a leftover hack from using helm2